client/src/controllers/SlugController.test.js

```diff
@@ -0,0 +1,40 @@
+import { Application } from '@hotwired/stimulus';
+import { SlugController } from './SlugController';
+
+describe('SlugController', () => {
+  beforeEach(() => {
+    document.body.innerHTML = `
+      <input
+        type="text"
+        name="slug"
+        aria-describedby="panel-child-promote-child-for_search_engines-child-slug-helptext"
```

Comment:
No need for the Aria attributes here. Best to keep unit test html to the bare minimum needed to produce the test output. 

Otherwise someone may think that the Aria attribute makes some part of the logic happen. In fact, in a future release of Stimulus the Aria elements that are set this way will be accessible in the controller and this test will likely fail if this line is left here & we upgrade.